webapp/src/components/properties/lastModifiedAt/__snapshots__/lastModifiedAt.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`componnets/properties/lastModifiedAt should match snapshot 1`] = `
```

Comment:
:open_mouth:  How did this test ever pass?

Reply:
The typo was in both sides, so... that is just an string, if the string is consistently wrong in bot sides, it pass.